frontend/packages/data-portal/app/components/I18n.tsx

```diff
@@ -26,6 +26,7 @@ export function I18n({ i18nKey, components, linkProps, ...props }: Props) {
         bold: <span className="font-bold" />,
         semibold: <span className="font-semibold" />,
         code: <span className="font-mono" />,
+        dash: <span className="border-b border-sds-gray-600 border-dashed" />,
```

Comment:
do you think we should merge this functionality with `<Link variant="dashed-underlined">`?